oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -166,6 +171,18 @@ public static GoogleCredentials fromStream(InputStream credentialsStream,
   }
 
   /**
+   * Saves the end user credentials in specified file
+   *
+   * @param credentials InputStream containing user credentials in JSON format
+   * @param credentialsFileName Name of file where to store the credentials
+   * @throws IOException An error saving the credentials.
+   */
+  public void saveCredentialsToFile(InputStream credentials, String credentialsFileName) throws IOException {
+    String userDirectory = System.getProperty("user.dir");
```

Comment:
What if the user wants to save the credentials at an absolute path, e.g. under "/tmp/..."?

Reply:
Would you like me to create several implementations or change this one for saving credentials file just at an absolute path?

Reply:
Change this to just not care - if the user gives relative path, then they get relative path; if they give an absolute path, they get an absolute path.